tests/integration/workflows/helpers/apps.py

```diff
@@ -0,0 +1,48 @@
+# -*- coding: utf-8 -*-
+#
+# This file is part of INSPIRE.
+# Copyright (C) 2014-2017 CERN.
+#
+# INSPIRE is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# INSPIRE is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with INSPIRE. If not, see <http://www.gnu.org/licenses/>.
+#
+# In applying this license, CERN does not waive the privileges and immunities
+# granted to it by virtue of its status as an Intergovernmental Organization
+# or submit itself to any jurisdiction.
+
+from __future__ import absolute_import, division, print_function
+
+import pytest
+
+from inspirehep.factory import create_app
+
+
+@pytest.fixture
+def workflow_app():
+    app = create_app(
+        BEARD_API_URL="http://example.com/beard",
+        DEBUG=True,
+        CELERY_ALWAYS_EAGER=True,
+        CELERY_RESULT_BACKEND='cache',
+        CELERY_CACHE_BACKEND='memory',
+        CELERY_EAGER_PROPAGATES_EXCEPTIONS=True,
+        PRODUCTION_MODE=True,
+        LEGACY_ROBOTUPLOAD_URL=(
+            'http://localhost:1234'
+        ),
+        MAGPIE_API_URL="http://example.com/magpie",
+        WTF_CSRF_ENABLED=False,
+    )
+
+    with app.app_context():
+        yield app
```

Comment:
This being a fixture makes no sense having it here, as it will not be loaded as such by pytest. Hmm... we should not make it a function anyhow, as we want to be able to use the `session` option in the fixture so it's loaded only once per run right? (it's quite slow...)